integration-tests/localconfig-example.properties

```diff
@@ -5,9 +5,6 @@
 
 # The following properties need to be updated to run the tests in locally
 
-webhook.site.uuid=webhook-uuid
-webhook.site.uuid.second=webhook-uuid-second
```

Comment:
nitpick here, maybe we can remove this line:

https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/dev/bin/localconfig-example#L24